.github/workflows/ios-unsigned.yml

```
@@ -133,6 +251,126 @@ jobs:
             cp build-log.txt "$OUTPUT_DIR_ABS/"
           fi
 
+      - name: Summarize xcodebuild log
+        if: ${{ always() }}
+        run: |
+          python3 <<'PY'
+from pathlib import Path
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 258: could not find expected ':'

(syntax-check)